### ISSUE TITLE
Who is Level 0 for?

### ISSUE THREAD

**Issue by mnot** [ietf/saa#14]
Right now, the first/default activity of the SAA is to quietly intervene by sending the person in question a private e-mail, with no public activity.

I think it's worth examining the thinking behind this.

ISTM it's attempting to be sensitive to the feelings of the person whose behaviour in question, giving them a chance to correct their behaviour without being publicly shamed.

While I don't think we should be trying to shame people, this approach leaves me wondering if we've considered the other effects of this policy -- namely that the target of the behaviour is left having to endure how they've been treated, and the rest of the community is left with the impression that the behaviour may or may not have be acceptable. 

In particular, if there isn't any visible reaction to unacceptable behaviour on the list, is there a risk that that effectively moves "the line" towards unacceptable behaviour?

Possible mitigations (not mutually exclusive in every case):

* Don't have a level 0; every incident is mentioned on-list, to give visibility
* Require some kind of [restorative justice](https://en.wikipedia.org/wiki/Restorative_justice) process to be followed
* Publish a weekly/monthly summary about how many incidents there were at each level

**Comment by sayrer** [comment]
I think Level 0 is the right approach. A preliminary intervention might require swift judgement, and since the SAAs are human, they could make mistakes.

Publishing a monthly summary seems like a good idea. That way, the SAAs can also record instances in which they overstepped, after some consideration.

**Comment by elear** [comment]
@mnot I appreciate your view, but I think it's off point from the goal, as I see it, which is to stop bad behavior in specific instances.  In my role in leadership, in the several times I have had to intervene, I did my level best to treat offenders as reasonable people.  Sometimes, they have just had a bad day, and sometimes it's clear that there is a deeper issue.  If private communication is enough to get results, then you have attained the goal without having had to resort to public shaming, which can often lead to an escalation (cf a certain head of state).

However, in cases where there is a clear target of bad behavior, it might make sense for the SAA to privately let any aggrieved party know that they have had a conversation with the individual, and are monitoring the situation.

**Comment by tfpauly** [comment]
Agreed with @elear’s point. For particularly bad cases, the SOP allows for skipping directly to level 1, so if there does need to be some notification to the list (because the violation is so egregious), that can be done.

This seems to leave level 0 for the “reasonable person who had a bad day/moment”. Perhaps the outlet for providing feedback to the list can be that the offender themselves sends a note/apology to the list at the recommendation of the SAA, giving them a chance to take responsibility more directly. If they aren’t willing to do that or continue in the behavior, things move to level 1.

**Comment by mnot** [comment]
I like the way you put that, @tfpauly. Might I suggest that description of the purpose of level 0 be modified with something that indicates it's for that case? I almost think it'd be better to say level 1 is the default, and level 0 is allowed at the discretion of the SAA...

I do think that we should contact the "target" of any behaviour when they're identifiable; perhaps we need a template for that, too.

Also, I think a regular report of how active the SAAs have been would be helpful. Weekly is probably too onerous; perhaps quarterly?

**Comment by elear** [comment]
Mark, I think we have a philosophical difference on how to handle these situations.  If I may try to characterize your position, it is that the best way to reassure the community that bad behavior won't be tolerated is to call it out as and when it occurs.  Would that be correct?

**Comment by mnot** [comment]
@elear no, although I think that position has merit, and we should at least be able to answer why we don't take it.

I think that it's a reasonable stance to allow things that the SAAs (subjectively) judge as "innocent" or "well-intentioned" or "just thoughtless" to be followed up "more gently", without embarrassing the offender, as Tommy said so well. Ultimately, this is likely to be more productive than actively shaming people (in their eyes) for minor lapses.

**However**, we should also account for the two impacts of this policy that I identified earlier:

1. The person who was the "target", if there was one, should know that there was followup, so they don't feel abandoned by the community. 
2. The community needs to know in some fashion that the SAAs *are* being active and not just letting things slide. This is in pretty direct conflict with the "not shaming" goal, so I suggested a monthly (or maybe quarterly?) report, just to show the community that the SAA is active. Not perfect, but I think worthwhile.

**Comment by elear** [comment]
Got it.  Ok, (much) less of a difference than I thought.
